lib/jdomodels/src/test/java/org/sagebionetworks/repo/model/dbo/dao/dataaccess/DBOSubmissionDAOImplTest.java

```diff
@@ -281,9 +314,74 @@ public void testListSubmissions() {
+		long modifiedOn = System.currentTimeMillis();
+		Submission dto1 = createSubmission(accessRequirement, researchProject, modifiedOn);
+		dtosToDelete.add( submissionDao.createSubmission(dto1).getSubmissionId() );
+		modifiedOn += 60000L;
+		submissionDao.updateSubmissionStatus(dto1.getId(), SubmissionState.APPROVED, null, user1.getId(), modifiedOn);
+			
+		// create a submission for research project 2
+		modifiedOn += 60000L;
+		Submission dto2 = createSubmission(accessRequirement, researchProject2, modifiedOn);
+		dtosToDelete.add( submissionDao.createSubmission(dto2).getSubmissionId() );	
+		modifiedOn += 60000L;
+		submissionDao.updateSubmissionStatus(dto2.getId(), SubmissionState.APPROVED, null, user1.getId(), modifiedOn);
+		
+		// now create another, later submission for research project 2
+		modifiedOn += 60000L;
+		Submission dto3 = createSubmission(accessRequirement, researchProject2, modifiedOn);
+		dtosToDelete.add( submissionDao.createSubmission(dto3).getSubmissionId() );	
+		modifiedOn += 60000L;
+		submissionDao.updateSubmissionStatus(dto3.getId(), SubmissionState.APPROVED, null, user1.getId(), modifiedOn);
+		SubmissionInfo dto3Info = createSubmissionInfo(researchProject2, modifiedOn);
+		
+		// now create another, later submission for research project 1
+		modifiedOn += 60000L;
+		Submission dto4 = createSubmission(accessRequirement, researchProject, modifiedOn);
+		dtosToDelete.add( submissionDao.createSubmission(dto4).getSubmissionId() );	
+		modifiedOn += 60000L;
+		submissionDao.updateSubmissionStatus(dto4.getId(), SubmissionState.APPROVED, null, user1.getId(), modifiedOn);
+		SubmissionInfo dto4Info = createSubmissionInfo(researchProject, modifiedOn);
+		
+		// create another submission for some other access requirement.  (Shouldn't see it in the results.)
+		modifiedOn += 60000L;
+		Submission dto5 = createSubmission(accessRequirement2, researchProject, modifiedOn);
+		dtosToDelete.add( submissionDao.createSubmission(dto5).getSubmissionId() );	
+		modifiedOn += 60000L;
+		submissionDao.updateSubmissionStatus(dto5.getId(), SubmissionState.APPROVED, null, user1.getId(), modifiedOn);
+		
+		// create a submission which is NOT approved.  (Shouldn't see it in the results.)
+		modifiedOn += 60000L;
+		Submission dto6 = createSubmission(accessRequirement, researchProject, modifiedOn);
+		dtosToDelete.add( submissionDao.createSubmission(dto6).getSubmissionId() );	
+		
+		// we should get back dto3 , then dto4, in that order
+		List<SubmissionInfo> expected = ImmutableList.of(dto3Info, dto4Info);
+
+		// method under test
+		List<SubmissionInfo> actual = submissionDao.listInfoForApprovedSubmissions(accessRequirement.getId().toString(), 10, 0);
+		
+		assertEquals(expected, actual);
+		
+		
+		// check that pagination works right:  If I get the second page of size *one*, I should just get dto4
+		actual = submissionDao.listInfoForApprovedSubmissions(accessRequirement.getId().toString(), 1, 1);
```

Comment:
Could you add a test to get a limit of 1 and offset of 0, and make sure you get dto3Info?